testing_config.py

```diff
@@ -89,3 +89,31 @@ def run(self, result=None):
     client = ndb.Client()
     with client.context():
       super(CustomTestCase, self).run(result=result)
+
+
+class Testdata(object):
+  def __init__(self, test_file_abs_directory, test_file_name):
+    """Helper class to load testdata
+    Common pattern to place the testdata in the following format:
+
+    Given a test file, atest_test.py, and it is located at
+    /some/module/atest_test.py.
+
+    The testdata should be located at /some/module/testdata/atest_test/
+    """
+    self.testdata = {}
+    self.testdata_dir = os.path.join(
+    test_file_abs_directory, 'testdata', test_file_name)
```

Comment:
Style: This continued line should be indented 4 spaces from the line above.  Maybe it is, but it does not look that way on GitHub.